core/src/main/java/org/elasticsearch/common/settings/AddStringKeyStoreCommand.java

```diff
@@ -64,6 +64,9 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         keystore.decrypt(new char[0] /* TODO: prompt for password when they are supported */);
 
         String setting = arguments.value(options);
+        if (setting == null) {
+            throw new UserException(ExitCodes.DATA_ERROR, "the setting name can not be null.");
```

Comment:
This should be a `USAGE` error, not a `DATA_ERROR` (and the period dropped from the exception message).